components/Grid.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native'
+import Dimensions from 'Dimensions'
+import Card from './Card'
+
+const { width, height } = Dimensions.get('window')
+
+export default class Grid extends Component {
+    render() {
+      const cards = this.props.grid.cardsInPlay
+        .map( (card, key) => <Card key={key} card={card} onTouch={this.props.touchCard} cardStyle={this.props.cardStyle }/> )
+      return (
+        <View style={gridStyle.gridBox}>
+          {cards}
+        </View>
+      )
+    }
+}
+
+
+
```

Comment:
I'm also noticing whitespace being used inconsistently - there was only a single line between the component and stylesheet in Card.js. Splitting the stylesheets into their own modules could be a good solution here.